src/Compilers/CSharp/Portable/BoundTree/BoundDecisionDagNode.cs

```diff
@@ -46,5 +51,104 @@ public override int GetHashCode()
                     throw ExceptionUtilities.UnexpectedValue(this);
             }
         }
+
+#if DEBUG
+        private int _id;
```

Comment:
Comments for differentiating uninitialized ID apply here too.

Reply:
Done